items/blind.lua

```diff
@@ -675,6 +675,40 @@ local scorch = {
 		end
 	end,
 }
+-- +0.25X blind requirements
+-- for every $5 you have when selected
+local greed = {
+	dependencies = {
+		items = {
+			"set_cry_blind",
+		},
+	},
+	object_type = "Blind",
+	name = "cry-greed",
+	key = "greed",
+	pos = { x = 0, y = 0 }, -- use Tax as placeholder icon
+	boss = {
+		min = 1,
+		max = 10,
+	},
+	atlas = "blinds",
+	order = 22,
+	boss_colour = HEX("4ca180"),
+	loc_vars = function(self, info_queue, card)
+		return { vars = { (get_blind_amount(G.GAME.round_resets.ante) * 0.25) } }
+	end,
+	collection_loc_vars = function(self)
+		return { vars = { localize("cry_greed_placeholder") } }
+	end,
+	set_blind = function(self, reset, silent)
+		G.GAME.blind.chips = (get_blind_amount(G.GAME.round_resets.ante) * 2 * G.GAME.starting_params.ante_scaling) + ((math.floor(G.GAME.dollars / 5)) * ((get_blind_amount(G.GAME.round_resets.ante) * 0.25))) -- go my equations
```

Comment:
please break this up into multiple lines, use the above card abilities, and support BigNums:
```suggestion
                -- go my equations
		G.GAME.blind.chips = 
		    lenient_bignum(
		        get_blind_amount(G.GAME.round_resets.ante) * 2 
		        * G.GAME.starting_params.ante_scaling
		    ) + (
		        (math.floor(G.GAME.dollars / to_big(self.center.abillity.extra.money_factor)) -- double check this is how you get the current cards center
		        * (get_blind_amount(G.GAME.round_resets.ante) * to_big(self.center.ability.extra.blind_mod))  -- double check this is how you get the current cards center
		    )
```

Reply:
If we cant rid the definition of magic numbers, we still should multi-line this. 

```suggestion
                -- go my equations
		G.GAME.blind.chips = lenient_bignum(
		    (
		        get_blind_amount(G.GAME.round_resets.ante) * 2 
		        * G.GAME.starting_params.ante_scaling
		    ) + (
		        (math.floor(G.GAME.dollars / 5))
		        * (get_blind_amount(G.GAME.round_resets.ante) * 0.25))
		    )
		)
```